app/services/credits/applied_coupon_service.rb

```diff
@@ -40,6 +42,12 @@ def already_applied?
     end
 
     def compute_amount
+      if applied_coupon.coupon.percentage?
+        discounted_value = invoice.amount_cents * (applied_coupon.percentage_rate.fdiv(100))
+
+        return discounted_value >= invoice.amount_cents ? invoice.amount_cents : discounted_value.round
```

Comment:
Style/TernaryParentheses: Use parentheses for ternary expressions with complex conditions.